jupyterhub/user.py

```diff
@@ -132,6 +134,20 @@ def __init__(self, orm_user, settings, **kwargs):
             config=self.settings.get('config'),
         )
 
+    @property
+    def get_spawners(self):
+        return self._instances
```

Comment:
I don't think we need to hide spawners behind a private attribute exposed as a public property. I think `user.spawners` can be the dict of Spawner instances